src/components/MainWrapper.js

```diff
@@ -14,10 +14,13 @@ const Wrapper = () => {
   wrapper.appendChild(upperSection);
   wrapper.appendChild(lowerSection);
 
+
```

Comment:
Takie trochę estetyczne ale jak się robi przerwy w kodzie to maksymalnie jednym enterem :D